{{cookiecutter.project_name}}/.flake8

```diff
@@ -2,33 +2,25 @@
 ignore =
     # whitespace before ':'
     E203,
-    # multiple spaces before operator
-    E221,
-    # multiple spaces after operator
-    E222,
-    # multiple spaces after ','
-    E241,
-    # unexpected spaces around keyword / parameter equals
-    E251,
-    # multiple spaces after keyword
-    E272,
-    # Missing docstring in public module
+    # too many leading ### in a block comment
+    E266,
+    # line too long (managed by black)
+    E501,
+    # Line break occurred before a binary operator (this is not PEP8 compatible)
+    W503,
+    # do not enforce existence of docstrings
     D100,
-    # Missing docstring in public class
     D101,
-    # Missing docstring in public method
```

Comment:
Why are we removing the comments here? As long as the ignore codes are here, I suggest keeping the comments as well

Reply:
That was a copy pasted .flake8 file... :wink: added again.